Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -186,6 +186,13 @@
           "SubId": "Php",
           "ContentsRegex": "$MySqlConnectionStringPhp",
           "MessageArguments": { "secretKind": "PHP MySQL connection string" }
+        },
+        {
+          "Id": "SEC101/114",
+          "Name": "DoNotExposePlaintextSecrets/PostgreSqlConnectionString",
+          "SubId": "ADO",
```

Comment:
>ADO [](start = 20, length = 3)

Casing here s/be 'Ado'. ADO is fine for your message args below, though. 'Ado' is the .NET casing standard for API.

Reply:
Fixed

---
In reply to: [568895337](https://github.com/microsoft/sarif-pattern-matcher/pull/157#discussion_r568895337) [](ancestors = 568895337)